recipes-core/packagegroups/packagegroup-prplmesh.bb

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+# SPDX-FileCopyrightText: 2020 the prplMesh contributors (see AUTHORS.md)
+# This code is subject to the terms of the BSD+Patent license.
+# See LICENSE file for more details.
+
+DESCRIPTION = "Packages needed to support prplMesh"
+
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COREBASE}/meta/COPYING.MIT;md5=3da9cfbcb788c80a0384361b4de20420"
+
+PR = "r0"
+
+inherit packagegroup
+
+RDEPENDS_${PN} = "\
+	prplmesh \
```

Comment:
Is this packagegroup really useful? It will be one package, always and forever. The package may pull in other dependencies (hostapd for example), but it will still be one package only from this layer.

Reply:
May be not right now but
1) there is such package group in Intel original `meta-prplmesh` layer
2) i'm planning to split prplmesh recipe into `prplmesh-agent` and `prplmesh-controller` to allow build RDKB for Turris Omnia as for extender as RDKB recommends
   So for some variants one or both that recipes will be present in that group

Reply:
I forgot that init scripts will be in separate recipes and depending on `DISTRO_FEATURES` one of next recipes will be a member of package group:
* prplmesh-systemd
* prplmesh-sysvinit